package.json

```diff
@@ -51,10 +51,10 @@
   ],
   "license": "MIT",
   "scripts": {
-    "test": "bun test src --coverage --bail --coverageSkipTestFiles=[test-utils.ts] --timeout 20000 && vitest run --typecheck",
```

Comment:
why?

Reply:
This was breaking the tests since it also included the examples, which even though all types are respected wouldn't work. I also wasn't able to exclude the text files in the vitest cli.